packages/contracts-bedrock/src/L2/L2ProxyAdmin.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
```

Comment:
Consider refactoring the `ProxyAdmin` to remove the L1 specific features and make a `L1ProxyAdmin`